deepchem/feat/base_classes.py

```diff
@@ -75,6 +76,60 @@ def _featurize(self, datapoint: Any):
     """
     raise NotImplementedError('Featurizer is not defined.')
 
+  def __repr__(self) -> str:
+    """Convert self to repr representation.
+
+    Returns
+    -------
+    str
+      The string represents the class.
+
+    Examples
+    --------
+    >>> import deepchem as dc
+    >>> dc.feat.CircularFingerprint(size=1024, radius=4)
+    CircularFingerprint[radius=4, size=1024, chiral=False, bonds=True, features=False, sparse=False, smiles=False]
+    >>> dc.feat.CGCNNFeaturizer()
+    CGCNNFeaturizer[radius=8.0, max_neighbors=8, step=0.2]
+    """
+    args_spec = inspect.getfullargspec(self.__init__)  # type: ignore
+    args_names = [arg for arg in args_spec.args if arg != 'self']
+    args_info = ''
+    for arg_name in args_names:
+      args_info += arg_name + '=' + str(self.__dict__[arg_name]) + ', '
+    return self.__class__.__name__ + '[' + args_info[:-2] + ']'
+
+  def __str__(self) -> str:
```

Comment:
This implementation is needed to resolve the Windows CI. This is referred #1829. __repr__ function shows all arguments when instantiating a class. __str__ function shows just updated arguments when instantiating a class.